packages/mobile-web/src/tools/mobile-offline/offline-analysis/ruleConfig.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2025, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import { CodeAnalysisBaseIssueType } from '../../../schemas/analysisSchema';
```

Comment:
I would like us to standardize on explicitly adding the `.js` extension to our import statements, for our local resources. I would actually like to ensure that we are a `type: "module"` (ESM) package, because there's really no reason to carry around the legacy of CommonJS in this day and age, and lots of reasons to abandon it. But I'm not sure we need to make that task a part of this PR.

At a minimum though, I want to be prepared with the proper ESM import statements with `.js` extensions, for our code imports.